tests/integration_tests/configs/default.jsonnet

```diff
@@ -4,13 +4,10 @@
     cmd: 'ethermintd',
     'start-flags': '--trace',
     config: {
-      consensus: {
-        // larger timeout for more stable mempool tests
-        timeout_commit: '2s',
-      },
       mempool: {
         // use v1 mempool to enable tx prioritization
         version: 'v1',
+        recheck: false,
```

Comment:
not sure if this hypass will hide any issue @yihuang

Reply:
disable recheck will increase the chance of including invalid tx into blocks, I guess it's not critical for tests, test case will check tx execution results anyway.